examples/batch/env.yaml

```diff
@@ -1,7 +1,7 @@
 dependencies:
 - pip:
-    - joblib>=1.2.0
+    - joblib>=0.13.0,<1.2.0  # >=1.2.0 upon upgrade of kserve's version
```

Comment:
Why are we allowing older version of joblib now?

Reply:
mainly because we haven't update kserve to use 1.2.0. For consistency (and non-breaking change) reason, I think we should use the older version first. But I think I will update the kserve first, so this line probably gets changed before it is merged.

Reply:
>we haven't update kserve to use 1.2.0

What kind of update are you referring to?

Reply:
We are deploying kserve using built image, hence the joblib there is frozen at whatever level was latest when the image was built. If we rebuild this time, it will be joblib 1.2.0. That will be a breaking (but necessary) change for our users using pyfunc and pyfunc v2.

In addition, sklearn server is tagged at 1.0.1 we should update that to >=1.1.2 so that users of python 3.8 is not limited by our version on kserve.

Reply:
But pyfunc and pyfunc v2 (batch prediction) doesn't use kserve image. So I'm not quite sure what's the purpose here. 
Additionally, even with currently new joblib version (joblib>=1.2.0) we didn't encounter any issue, so allowing the older version seems strange to me.

Reply:
That's correct. What I mean is that we should make it consistent. Otherwise users may be confused because when they use standard models, they have to use old joblib but when they use pyfunc and v2, they have to use 1.2.0

Reply:
I got your point, but even for the standard model we already use joblib>=1.2.0 (https://github.com/gojek/merlin/blob/0ab987ab6a1629c74d3e91a82a6b6c62b86e2c02/python/sdk/setup.py#L51) I wonder from where do you get `joblib>=0.13.0`

Reply:
no, that was updated by me 9 days ago. Unfortunately it won't work for standard model, that's how I discovered it is controlled by kserve on the server side.

Got the version from here:
https://github.com/kserve/kserve/blob/master/python/sklearnserver/setup.py#L37

Reply:
Ah ok, but then how did it pass our end to end test?

Reply:
Because our test uses pre-saved models instead of saving new ones with joblib

Reply:
Make sense, thanks!